rundeckapp/grails-spa/packages/ui-trellis/src/app/components/readme-motd/editProjectFileService.ts

```diff
@@ -0,0 +1,69 @@
+import {getRundeckContext} from "../../../library";
+import {Notification} from "uiv";
+
+const rundeckClient = getRundeckContext().rundeckClient
+function notifySuccess(title: string, message: string) {
+    Notification.notify({
+        type: "success",
+        title: title,
+        content: message,
+        duration: 5000
+    });
+}
+
+function notifyError(message: string) {
+    Notification.notify({
+        type: "danger",
+        title: "An Error Occurred",
+        content: message,
+        duration: 0
+    });
+}
+
+function notifyWarning(message: string) {
+    Notification.notify({
+        type: "info",
+        title: "Warning",
+        content: message,
+        duration: 0
+    });
+}
```

Comment:
The idea is not to move the notify calls to the service file, just the api call itself. So all methods in the service file return a response or an error.
Because then if one day other components need to use these API calls but do something else based on the response, it's possible to reuse these methods in the service